ci/requirements/linux_full_deps_conda.txt

```diff
@@ -16,7 +16,7 @@ pysdl2
 pytest
 pytest-cov
 pytest-sugar
-pyopengl
+pyopengl!=3.1.7
```

Comment:
If I understood @aganders3 correctly, this will start failing when 3.1.8 or 3.1.9 is released on conda-forge. Do we want to do `=3.1.6` here?

Reply:
It depends on what people from pyopengl do. Maybe they fix this. 

My personal preference is to just block the problematic version. But it works much better in active project where you have multiple CI job per day, so you could easilly diff what happens in the environment. 

So it is a question, with which solution you will feel better?

Reply:
pyopengl on conda-forge updating by bot is broken:
https://conda-forge.org/status/